notebooks/lightning_segmentation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>change <code> for some reason </code>to <code>for these reasons  :) </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PhoenixDL/rising/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>